tests/acceptance/features/apiNotification/deprovisioningNotification.feature

```diff
@@ -0,0 +1,172 @@
+          "messageRichParameters": {
+            "type": "object"
+          },
+          "notification_id": {
+            "type": "string",
+            "enum": [
+              "deprovision"
+            ]
+          },
+          "object_id": {
+            "type": "string"
+          },
+          "object_type": {
+            "type": "string",
+            "enum": [
+              "resource"
+            ]
+          },
+          "subject": {
+            "type": "string",
+            "enum": [
+              "Instance will be shut down and deprovisioned"
+            ]
+          },
+          "subjectRich": {
+            "type": "string",
+            "enum": [
+              "Instance will be shut down and deprovisioned"
+            ]
+          },
+          "user": {
+            "type": "string"
+          }
+        }
+      }
+      """
+
+
+  Scenario Outline: non-admin user cannot create a deprovisioning notification
+    Given the administrator has assigned the role "<role>" to user "Alice" using the Graph API
+    When user "Alice" tries to create a deprovisioning notification
+    Then the HTTP status code should be "404"
+    And user "Alice" should not have any notification
+    Examples:
+      | role        |
+      | Space Admin |
+      | User        |
+      | User Light  |
+
+
+  Scenario Outline: get a deprovisioning notification in various languages
+    Given admin has created a deprovisioning notification
+    And user "Alice" has switched the system language to "<language>"
+    When user "Alice" lists all notifications
+    Then the HTTP status code should be "200"
+    And the JSON response should contain a notification message with the subject "<subject>" and the message-details should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "message"
+        ],
+        "properties": {
+          "message": {
+            "type": "string",
+            "enum": [
+              "<message>"
+            ]
+          }
+        }
+      }
+      """
+    Examples:
+      | language | subject                                                          | message                                                                                                                                                                                                 |
+      | de       | Instanz wird heruntergefahren und außer Betrieb genommen werden. | Achtung! Diese Instanz wird am 2043-07-04T11:23:12Z heruntergefahren und außer Betrieb genommen werden. Laden Sie Ihre Daten vor diesem Tag herunter, da Sie danach nicht mehr darauf zugreifen können. |
+      | es       | La instancia se cerrará y se desaprovisionará                    | ¡Atención! La instancia se cerrará y se desaprovisionará el 2043-07-04T11:23:12Z. Descarga todos tus datos antes de esa fecha, puesto que el acceso pasada la fecha no será posible.                    |
+
+
+  Scenario: after reading the notice, it reappears again
```

Comment:
Is this scenario trying to say that deleting the normal notification i.e sending api request to the `/apps/notifications/api/v1/notifications?format=json` won't delete the deprovisioning notification?

If that's the case, would this make sense
```
```suggestion
  Scenario: try to delete deprovisioning notification through the normal notification endpoint 
```

Reply:
I think, the scenario wants to tell that even notifications are marked as read, the deprovisioning notification will not get deleted - reappears again

Reply:
But isn't `marking as read` `Deleteing` the notification, I think it sends the delete request but I might be wrong

Reply:
yes, it is

Reply:
`deprovisioning notification reappears again even after being marked as read` is a good name. thanks +1